acasclient/selfile.py

```diff
@@ -682,15 +759,17 @@ def raw_results(self, new_dict):
     def raw_results_datatype(self):
         """OrderedDict {(column1: temp_id1), ...)} column name keys for raw results data."""
         return self._raw_results
+
     @raw_results.setter
-    def raw_results(self, new_dict):
+    def raw_results_datatype(self, new_dict):
```

Comment:
I'm confused by this function and its new name.
Do we have two `@raw_results.setter` functions? One on this line and another on line 754 above?
It seems like prior to your change we just had duplicated code. After your change we have a function named `raw_results_datatype` that is annotated as a setter for the `raw_results` property, not the `raw_results_datatype` property... and the code for both updates the `self._raw_results` dictionary.

Reply:
This looks better.